modules/google-speech/src/backend/client.ts

```diff
@@ -81,7 +81,12 @@ export class GoogleSpeechClient {
   }
 
   public async speechToText(audioFileUrl: string, language: string, timeout?: string): Promise<string | undefined> {
```

Comment:
This should come with unit tests, there's a lot of tricky custom logic

Reply:
Totally agree! I will do it in another PR tho!